labgrid/protocol/consoleprotocol.py

```diff
@@ -1,11 +1,14 @@
 import abc
+from typing import Optional
+
+from pexpext import EOF, TIMEOUT
```

Comment:
This seems to be a typo breaking the CI?

```suggestion
from pexpect import EOF, TIMEOUT
```